docs/en/Clients-S3.md

```diff
@@ -158,6 +158,91 @@ Server: Jetty(9.2.z-SNAPSHOT)
 # curl -i -X DELETE http://localhost:39999/api/v1/s3/testbucket
 HTTP/1.1 204 No Content
 Date: Tue, 29 Aug 2017 22:45:19 GMT
+```
+
+### Initiate a multipart upload
```

Comment:
nit: can you put the multi-part upload doc before `Delete an empty bucket` ? so when user walk through and follow the doc, they won't run into the bucket not exist error.
Feel free to do this in a follow-up PR, thanks

Reply:
same for the python part

Reply:
ditto for the python script part

Reply:
done in https://github.com/Alluxio/alluxio/pull/6046